sos/plugins/tomcat.py

```diff
@@ -13,6 +13,7 @@
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
 from sos.plugins import Plugin, RedHatPlugin
+from datetime import datetime
```

Comment:
Not really a review comment, more a note-to-self: we have a number of plugins now doing time based calculations (tomcat, sytemd, pacemaker) so I've filed  #838 to keep track of that (there's no need for that work to block this pull request however).